docs/sources/reference/commandline/cli.md

```diff
@@ -482,10 +482,13 @@ simultaneously, screen sharing style, or quickly view the progress of your
 daemonized process.
 
 You can detach from the container (and leave it running) with `CTRL-p CTRL-q`
-(for a quiet exit) or `CTRL-c` which will send a `SIGKILL` to the container.
+(for a quiet exit) or `CTRL-c` which will send a `SIGINT` to the container.
```

Comment:
This paragraph structure is a bit wonk. If you could please clean it up while you are in there.  Also the note needs reformatting as @thaJeztah notes.

Markdown here: https://gist.github.com/moxiegirl/60a935fccb2e33c8e5ff

---> suggested replace

You can detach from a container and leave it running with `CTRL-p CTRL-q`. This
is called a quiet exit. You can also exit using `CTRL-c` which sends a
`SIGINT` to the container. When you are attached to a container and exit its
main process, Docker returns the process' exit code to the client.

>**Note**: A process running as PID 1 inside a container does not automatically
>terminate on receipt of `SIGINT` or `SIGTERM`.